src/main/resources/org/jenkinsci/plugins/github/config/GitHubPluginConfig/config.groovy

```diff
@@ -5,15 +5,12 @@ import com.cloudbees.jenkins.GitHubPushTrigger
 def f = namespace(lib.FormTagLib);
 
 f.section(title: descriptor.displayName) {
-    f.entry(title: _("Servers configs with credentials to manage GitHub integrations"),
-            description: _("List of GitHub Servers to manage hooks, set commit statuses etc."),
```

Comment:
In the interest of getting other changes in, if this is something you feel strongly I'm happy to give up, but  you should see it when you have one or two `GitHubServerConfig` instances. This text will really look out of place. Existing help file should explain what this field does. In addition, if this plugin is to be better reused from elsewhere, the wording is incorrect, too.